python/kfserving/kfserving/test_storage.py

```diff
@@ -50,6 +50,13 @@ def test_mock_gcs(mock_storage):
     mock_storage.Client().bucket().list_blobs().__iter__.return_value = [mock_obj]
     assert kfserving.Storage.download(gcs_path)
 
+@mock.patch(STORAGE_MODULE + '.BlockBlobService')
+def test_mock_blob(mock_storage):
+    blob_path = 'https://accountname.blob.core.windows.net/container/blob'
```

Comment:
can add a few negative tests too?
also one that requires a directory to get created?